web_root/js/sockets.js

```diff
@@ -12,7 +12,7 @@
         var container = $("#content");
         var domain = window.location.host;
 
-        var socket = new WebSocket("ws://" + domain + "/ws");
+        var socket = new WebSocket("ws://" + domain + ":8080/ws");
```

Comment:
Oh, I know why. I'm pulling in the test web page over a different webserver (nginx), so obviously it's not going through the Node webserver's port.